### PR TITLE
DP-15813 MF Refine campaign video component v 0.2

### PR DESCRIPTION
**Pull request by lannyheidbreder-current**
## Description
Added new color and background options, and made other style tweaks to match the spec and mockups.

## Related Issue / Ticket

- [JIRA issue](https://jira.mass.gov/browse/DP-15813)

#### Go here and do this

Visit `?p=organisms-campaign-video` and `?p=organisms-campaign-video-not-top` in Mayflower

#### What you should see

Both components match the mockups and spec, including the styles' default color schemes.

## Screenshots

Screen Shot 2019-09-13 at 9 07 34 AM
Screen Shot 2019-09-13 at 9 07 19 AM

#### Impacted Areas in Application

- campaign-video component
- Other components using it, e.g. `?p=pages-marketing-campaign`

### CODE REVIEW

**Review by clair0917 — APPROVED** [review]
looks great

File: patternlab/styleguide/source/_patterns/03-organisms/by-template/campaign-video~not-top.json [hunk 1]
```diff
@@ -1,10 +1,8 @@
...
     "id": "2",
-    "positionTop": "",
-    "backgroundColor": "bay-blue",
-    "textColor": "white",
-    "title": "Why work for the commonwealth",
+    "isHeader": false,
+    "title": "Why work for the commonwealth?",
     "description": "Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur.",
     "video": {
       "src": "https://www.youtube.com/embed/dEkUq-Rs-Tc",
```
Comment (ygannett) [inline patternlab/styleguide/source/_patterns/03-organisms/by-template/campaign-video~not-top.json:8]: Please add `label` to enable to add `aria-label` to the iframe in **video.twig**.  `aria-label` value should have the video title. So, assistive technology users can know what's the video about. `aria-label` is supposed to be `title`, but I'll open another ticket to fix this.
FYI, the **video.json** is shown in the above comment.

File: patternlab/styleguide/source/_patterns/03-organisms/by-template/campaign-video.json [hunk 2]
```diff
@@ -1,9 +1,8 @@
...
   "campaignVideo": {
     "id": "abc1",
-    "backgroundColor": "",
-    "textColor": "",
-    "title": "Why work for the commonwealth",
+    "isHeader": true,
+    "title": "Why work for the commonwealth?",
     "description": "Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur.",
     "video": {
       "src": "https://www.youtube.com/embed/dEkUq-Rs-Tc",
```
Comment (ygannett) [inline patternlab/styleguide/source/_patterns/03-organisms/by-template/campaign-video.json:8]: Please add `label` to enable to add `aria-label` to the iframe in **video.twig**.  `aria-label` value should have the video title. So, assistive technology users can know what's the video about. `aria-label` is supposed to be `title`, but I'll open another ticket to fix this.
FYI, here is the **video.json**:
```
{
  "video": {
    "src": "https://www.youtube.com/embed/dEkUq-Rs-Tc",
    "label": "Using a gas grill safely",
    "width": "853",
    "height": "480",
    "link": {
      "href": "/patterns/05-pages-GEN-Building-Energy-Codes/05-pages-GEN-Building-Energy-Codes.html",
      "text": "View transcript",
      "info": "View transcript of Using a gas grill safely",
      "property": ""
    },
    "position": ""
  }
}
```

**Review by ygannett — APPROVED** [review]
Thank you for the change.
LGTM.

File: patternlab/styleguide/source/_patterns/03-organisms/by-template/campaign-video.json (hunk 2 above)
Comment (clair0917) [reply patternlab/styleguide/source/_patterns/03-organisms/by-template/campaign-video.json:8]: @ygannett 
please spin up a new ticket to add a11y accessibility to current components.

**Pull request merged**